ubuntu/python-conda/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM ubuntu:18.04 as builder
```

Comment:
will we need to update this once 9.0 is out? And will this need to be backwards compatible?

Reply:
18.04 on DCS works with 9.x clusters, since it's replacing the base layers. See https://e2-dogfood.staging.cloud.databricks.com/?o=6051921418418893#setting/clusters/0703-185620-tenet65/configuration as an example cluster that uses this image with DBR 9.x. 

We could replace this with 20.04, but I don't think it's necessarily required.

Reply:
oh awesome, thanks for clarifying!

Reply:
Could it be updated to 20.04? I need this version of ubuntu and was expecting it in DBR9.x
https://github.com/databricks/containers/issues/63